### ISSUE TITLE
blacklist request

### ISSUE THREAD

**Issue by vatsalchandan** [AdguardTeam/AdGuardDNS#1]
ad.mi.com
ad.xiaomi.com

If possible block these two domains

**Comment by luxysiv** [comment]
Add this domain 

ng-vn-notice.gameitop.com

**Comment by GulnazSalakhova** [comment]
Thanks for taking the time to report this. These domains are blocked by the [AdGuard DNS filter blocklist](https://adguardteam.github.io/AdGuardSDNSFilter/Filters/filter.txt).  If you think that some issue from your problem has not been closed, you can open a new issue.
